Documentation/Browser/AddNewIDLFile.md

```diff
@@ -28,7 +28,7 @@ interface CSSRule {
     - `#include <LibWeb/Bindings/HTMLDetailsElementPrototype.h>` to the includes list.
     - `ADD_WINDOW_OBJECT_INTERFACE(HTMLDetailsElement)      \` to the macro at the bottom.
 
-4. Add a `libweb_js_wrapper(HTML/HTMLDetailsElement)` call to [`LibWeb/CMakeLists.txt`](../../Userland/Libraries/LibWeb/CMakeLists.txt)
+3. Add a `libweb_js_wrapper(HTML/HTMLDetailsElement)` call to [`LibWeb/idl_files.cmake`](../../Userland/Libraries/LibWeb/idl_files.cmake)
```

Comment:
Why change the step number here?

Reply:
Oh, good catch, I didn't mean to change that. It's just that the number were changed inbetween the time I made the commit and the numerous rebases, and I didn't notice it.
Thanks for the catch, pushed the fix :)